src/spectrecoff/Spectre/Prompt.fs

```diff
@@ -1,37 +1,59 @@
 [<AutoOpen>]
 module SpectreCoff.Prompt
 
-open Spectre.Console 
+open Spectre.Console
 
-type PromptOptions = 
+type PromptOptions =
     { Secret: bool
       Optional: bool }
 
-type MultiSelectionPromptOptions = 
+type MultiSelectionPromptOptions =
     { PageSize: int }
 
-let mutable defaultOptions = 
+type ChoiceGroup<'T> =
+    { Group: 'T
```

Comment:
Why does the group have to be a 'T? Isn't it just a title and not an item in of itself?

Reply:
I found this confusing too, and mentioned it in the documentation. This is how it is in the Spectre api. 
In any case I was planning to approach you about a better solution, stay tuned.